### PR TITLE
update to paper-styles#0.8.2

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata PTAL

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -21,14 +21,14 @@
   ],
   "dependencies": {
     "polymer": "Polymer/polymer#0.8-preview",
-    "layout": "Polymer/layout",
-    "paper-styles": "PolymerLabs/paper-styles"
+    "paper-styles": "PolymerElements/paper-styles"
```
Comment (cdata) [inline bower.json]: Is it possible to depend on `polymerelements/paper-styles#^0.8.0`? This way we don't have to automatically update it again until `0.9.0` lands.

File: bower.json [hunk 2]
```diff
@@ -21,14 +21,14 @@
...
   },
   "devDependencies": {
     "webcomponentsjs": "Polymer/webcomponentsjs",
     "core-doc-viewer": "Polymer/core-doc-viewer#0.8-preview",
     "core-icon": "Polymer/core-icon#0.8-preview",
     "core-icons": "Polymer/core-icons#0.8-preview",
+    "test-fixture": "Polymer/test-fixture#master",
```
Comment (cdata) [inline bower.json]: You can currently use:

`polymerelements/test-fixture#^0.8.0`

And I have some PRs up (you are assigned) for other elements including the following:

```
polymerelements/iron-icon#^0.8.0
polymerelements/iron-icons#^0.8.0
```

**Comment by cdata** [comment]
Is this PR still relevant @morethanreal ?

**Comment by morethanreal** [comment]
I don't think so.

**Pull request closed without merging**